frontend/src/types/RegistrationTypes.ts

```diff
@@ -0,0 +1,6 @@
+export type CartItem = {
```

Comment:
Why do we need a CartItem type? I thought we could use the array of camper objects to determine these fields? Or was this just a temporary work-around?

Reply:
Yeah I'll prob map directly eventually, this was a nice in between for dummy props